api/all.yaml

```diff
@@ -29,6 +29,9 @@ tags:
     description: |
       Model versions contain all artifact files associated with an individual machine learning model. They are associated with a single parent model and have an integer identifier, indicating the version number of the model.
       Model versions are referenced via model URI's using the Unity Catalog three-level namespace of the registered model and the version identifier.
+  - name: Permissions
```

Comment:
nit: Is this for this PR or should it be moved to the access conrol PR?

Reply:
yep. good catch. missed that when I extracted the user database stuff from the access control work.